packages/notifier/README.md

```diff
@@ -1,48 +1,59 @@
-# Notifier
+# Notifiers and Subscriptions
 
-This package provides two similar abstractions for producing and consuming
-asyncronous sequences of values, the *NotifierKit* and the *SubscriptionKit*.
-Both enable a service to notify clients of state changes.
+This package provides two similar abstractions for producing and consuming asynchronous value 
+sequences, the *NotifierKit* and the *SubscriptionKit*. Both let a service notify clients of state changes.
 
-For user documentation on the NotifierKit, see the
-[documentation](https://agoric.com/documentation/distributed-programming.html#notifiers).
-The following is a more precise description of the semantics of the
-NotifierKit, the SubscriptionKit, and their distributed system properties.
+In JavaScript, async iterations are manipulated by `AsyncGenerators`, `AsyncIterables`, and `AsyncIterators`. 
+For an introduction to these concepts and implementations, 
+see [here](https://javascript.info/async-iterators-generators).
 
+**Important**: In this doc, a *non-final value* is **not** a reference to JavaScript’s `final` keyword. 
```

Comment:
Removed it from here, put a short bit (without an **Important**) in the body noting this to avoid similar confusion by others (darn Google search; I did a search on "JavaScript final value" to be sure, and didn't notice it'd only returned results for "Java final value"....)